library/src/commonMain/kotlin/ovh/plrapps/mapcompose/utils/IODIspatcher.kt

```diff
@@ -0,0 +1,5 @@
+package ovh.plrapps.mapcompose.utils
```

Comment:
Not very important, but this file and the others should be renamed `IODispatcher...`